vignettes/evals.Rmd

```diff
@@ -0,0 +1,180 @@
+---
+title: "Evals"
+author: "Roman Tsegelskyi "
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Evals}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, echo = FALSE, message = FALSE}
+knitr::opts_chunk$set(collapse = T, comment = "#>")
+library(pander)
+```
+
+`evals` is aimed at collecting as much information as possible while evaluating R code. It can evaluate a character vector of R expressions, and it returns a list of captured information while running those:
+
+* `src` holds the R expression,
+* `result` contains the raw R object as is,
+* `output` represents how the R object is printed to the standard output,
+* `type` is the class of the returned R object,
+* `msg` is a list of possible messages captured while running the R expression and
```

Comment:
Although warnings and errors are kind of messages, but because `message` is an R message type, I'd also mention warnings and errors here explicitely